app/frontend/tokenRegistry/tokenRegistryApi.ts

```diff
@@ -0,0 +1,46 @@
+import request from '../wallet/helpers/request'
+import {ADALITE_CONFIG} from '../config'
+import {RegisteredTokenMetadata} from '../types'
+import {TokenRegistryApi} from './types'
+
+export default (): TokenRegistryApi => {
+  const fetchTokensMetadata = (subjects: string[]): Promise<any> => {
+    const url = `${ADALITE_CONFIG.ADALITE_SERVER_URL}/api/tokenRegistry/getTokensMetadata`
```

Comment:
to be consistent with our adalite backend's bulk calls, I'd consider making the route `api/bulk/tokens/metadata`

Reply:
Refactored